olp-cpp-sdk-core/tests/cache/DefaultCacheTest.cpp

```diff
@@ -219,3 +220,57 @@ TEST(DefaultCacheTest, AlreadyInUsePath) {
   olp::cache::DefaultCache cache2(settings);
   ASSERT_EQ(olp::cache::DefaultCache::OpenDiskPathFailure, cache2.Open());
 }
+
+class ProtectedCacheTest : public ::testing::Test {
```

Comment:
Why do you need a separate test fixture? Please include test into DefaultCacheTest fixture. Protected/mutable caches are both part of the same class and should not be handled separately but together!

Reply:
The tests for protected cache requires the setup (they need the prepared valid cache). And since more tests will come in the future (to test Get) I thought it would be better to add test fixture.

Reply:
Or I can probably make DefaultCacheTest a test fixture. What do you think?